jormungandr/Cargo.toml

```diff
@@ -36,10 +36,11 @@ hex = "0.4"
 http = "0.2.2"
 humantime = "2.0"
 jormungandr-lib = { path = "../jormungandr-lib" }
+keynesis = "1.1"
 lazy_static = "1.4"
 linked-hash-map = "0.5"
-poldercast = "0.14"
-multiaddr = { package = "parity-multiaddr", version = "0.9" }
+poldercast = { git = "https://github.com/Zeegomo/poldercast", branch = "master" }
```

Comment:
No they are all released, following prs points to the upstream repo. This points to the fork because since topology does not live on its own task yet we need to add `Send + Sync` bounds that are not required once make this change.

Reply:
Does poldercast 1.2 fix this?

Reply:
Nope, that change was never proposed, we solved that by moving topology to its own task